c-boyle/future-hero
# Miscellaneous Level 1 Changes (volumetric lighting, particle effects, etc)

## Changes:
- Add volumetric lighting to spotlights.
- Add music note particle effect to boombox.
- Add angry emotion particle effect to ruffians and smoker when they exit the restaurant.
- Add `TimedAudio` component to boombox music to make it speed up when time is running out.
- Make boombox disable its intractability (and lose its outline) after use.

## Screenshots:

Cigarette spotlight from the door:
![image](https://user-images.githubusercontent.com/62191940/198930323-ec387806-45e2-4022-a3b9-19afdd037311.png)

Cigarette spotlight up close:
![image](https://user-images.githubusercontent.com/62191940/198930409-b87c6de6-f5bf-4b3b-9e30-9c71b065e874.png)

Old lady and ruffians spotlight:
![image](https://user-images.githubusercontent.com/62191940/198930446-0bfe54fb-9df9-41e9-9f74-5960374043af.png)

Old lady puddle spotlight:
![image](https://user-images.githubusercontent.com/62191940/198930650-00331669-2f21-48e3-8afd-65ff49af2e18.png)

Boombox with music notes:
![image](https://user-images.githubusercontent.com/62191940/198930483-198db16d-55b9-4ead-a7cb-78d86ab89b60.png)

Angry smoker:
![image](https://user-images.githubusercontent.com/62191940/198930520-f3aead0d-5ead-496e-a441-098e8fe16cdd.png)

Angry ruffians:
![image](https://user-images.githubusercontent.com/62191940/198930564-5757db6a-122a-43bb-83be-34765ffbeaaa.png)

### How to Test:
Open the scene, check out the spotlights in future vision, solve the challenges to see people get angry.
